nexus/tests/output/unexpected-authz-endpoints.txt

```diff
@@ -1,3 +1,4 @@
 API endpoints tested by unauthorized.rs but not found in the OpenAPI spec:
 PUT    "/v1/system/update/repository?file_name=demo-repo.zip"
 GET    "/v1/system/update/repository/1.0.0"
+DELETE "/v1/webhooks/receivers/my-great-webhook/subscriptions/test.foo.**"
```

Comment:
this feels not on purpose

Reply:
oh, i think i may need to rerun that test after regenerating the spec, good catch